x-pack/solutions/observability/plugins/observability_onboarding/public/application/app.tsx

```diff
@@ -53,10 +52,6 @@ export function ObservabilityOnboardingAppRoot({
     context,
   };
 
-  core.analytics.reportEvent(OBSERVABILITY_ONBOARDING_TELEMETRY_EVENT.eventType, {
-    uses_legacy_onboarding_page: false,
-  });
```

Comment:
Deleting this one as legacy onboarding page is not a thing anymore. Also deleted `uses_legacy_onboarding_page` property from the even schema.